app/Http/Requests/Backend/UpdateUserRequest.php

```diff
@@ -23,7 +24,7 @@ public function authorize()
     public function rules()
     {
         return [
-            'password' => 'required|min:3',
+            'password' => 'nullable|min:3',
```

Comment:
default, `password` is not required, so `null` it ok. If have input, this will check `min:3` . You can check when remove this :).